aea/crypto/cosmos.py

```diff
@@ -447,6 +621,25 @@ def _try_get_account_number_and_sequence(
             )
         return result
 
+    def send_signed_cosmwasm_transaction(
```

Comment:
This should be called from `send_signed_transaction`. In `send_signed_transaction` you need to identify whether to call this or continue with old approach based on signed message.

Reply:
it might be a bit tricky because right now I am running wasmcli in block mode which means that I get receipt/timeout in one function call. If I get it right the original approach is to non-blocking broadcast, wait, get_receipt rather than blocking broadcast which gives you a receipt when it's done.